app/src/main/java/com/onboarding/parkingsystemjava/mvp/presenter/ConfigureParkingPresenter.java

```diff
@@ -0,0 +1,33 @@
+package com.onboarding.parkingsystemjava.mvp.presenter;
+
+import com.onboarding.parkingsystemjava.listener.OnInputListener;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+import com.onboarding.parkingsystemjava.utils.ConstantUtils;
+
+public class ConfigureParkingPresenter implements SetParkingContract.SetParkingPresenter {
+    private final SetParkingContract.SetParkingView view;
+    private final OnInputListener inputListener;
+
+    public ConfigureParkingPresenter(OnInputListener inputListener, SetParkingContract.SetParkingView view) {
+        this.view = view;
+        this.inputListener = inputListener;
+    }
+
+    @Override
+    public void OnOkButtonPress() {
+        String lots = view.getLots();
+        if (!lots.equals(ConstantUtils.EMPTY_STRING)) {
```

Comment:
maybe here you can use the lots.isEmpty() instead of the equals comparison

Reply:
you could told me this before I made the empty string constant

Reply:
done